maquetado/books-list.html

```diff
@@ -0,0 +1,57 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Books list</title>
+  <meta name="description" content="books-list">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+  <div class="big-container">
+    <div class="books-list-container">
```

Comment:
Then we could write this instead

```suggestion
    <div class="books-list">
```